lib/gds-sso/config.rb

```diff
@@ -3,6 +3,8 @@
 module GDS
   module SSO
     module Config
```

Comment:
Could we use `Singleton` here to avoid class vars?

Reply:
I'm not sure, as this is a module not a class.

Reply:
I think it could be converted to a class quite easily, but I also understand that's a bit out-of-scope.